src/Collection.php

```diff
@@ -219,6 +234,10 @@ public function aggregate(array $pipeline, array $options = [])
             $options['readConcern'] = $this->readConcern;
         }
 
+        if (! array_key_exists('codec', $options)) {
```

Comment:
I understand that `array_key_exists` is used to be able to remove the default codec option.